source/common/ssl/context_manager_impl.cc

```diff
@@ -20,18 +20,19 @@ void ContextManagerImpl::releaseContext(Context* context) {
   contexts_.remove(context);
```

Comment:
update the comment above about rename if we do the rename.

Reply:
Reverted it. Please see my other comment